screens/LuresResults.js

```diff
@@ -1031,6 +1210,23 @@ elevation: 3,
 
   return (
     <ScrollView style={{ flex: 1, backgroundColor: 'white' }}>
+      {reactIfView(currentTutorial == 'ResultsGuide', <Tooltip
+        contentStyle={tutorial_styles.doubleLine}
+        backgroundColor={tutorial_transparent}
+        isVisible={currentTutorial == 'ResultsGuide'}
+        content={<Text style={[text_style.fontColorWhite]}>{loadTranslations("tutGuide")}</Text>}
+        placement="bottom"
+        onClose={async () => {setCurrentTutorial(await updateTutorialAndGetNext('ResultsGuide', NAV_LURES_RESULTS))}}
+      >
+        <TooltipChildrenContext.Consumer>
+          {({ tooltipDuplicate }) => (
+            reactIfView(!tooltipDuplicate,
+              <View style={[{height:Platform.OS === 'android' ? 50 : 0, width: width}]}></View>
+            )
```

Comment:
This condition repeats itself throughout the entire file, try to make a method for it. We try to avoid duplicate code as much as possible.

Reply:
So I fixed this repetition in the Conditions Results MR, and that branch was made from this lure results branch here. Once those are merged, and lure-results-tutorials has the TutorialTooltip file, I'll adjust LuresResults.js to use that.